packages/mapviewer/src/utils/composables/useMovableElement.composable.js

```diff
@@ -24,8 +23,18 @@ const MovementSource = Object.freeze({
  * @param {string[]} [options.initialPositionClasses] Initial position classes to remove when the
  *   element is being moved
  */
-export function useMovableElement(element, options = {}) {
-    const { grabElement = null, offset = null, initialPositionClasses = [] } = options
+export function useMovableElement(config = {}) {
+    const {
+        element = null,
+        grabElement = null,
+        offset = null,
+        initialPositionClasses = [],
+    } = config
+
+    if (!config.element || !config.grabElement) {
+        throw new Error('Element/grabElement is required')
```

Comment:
I would make the error a bit more explicit : 

`[useMovableElement] Element, grabElement and offset are required to use a movable element. Received ${element} for element, ${grabElement} for grabElement and ${offset} for offset.` or something like that.